newIDE/app/src/Utils/UpdateDiscordRichPresence.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import optionalRequire from './OptionalRequire';
+const electron = optionalRequire('electron');
+const ipc = electron ? electron.ipcRenderer : null;
+
+/*::
+type Params = {
+  project?: gdProject,
+  resetDate?: boolean,
+};
+*/
+
+const updatePresence = (params /*: ?Params */) => {
+  if (ipc === null) return;
+
+  params = params || {};
+  const { project, resetDate } = params;
+
+  if (resetDate === true) updatePresence.date = Date.now();
+  updatePresence.date = updatePresence.date || Date.now();
+
+  let state = {
+    details: 'Making a game with GDevelop.',
+    state: 'Not working on any game.',
+    startTimestamp: updatePresence.date,
+    largeImageKey: 'gdicon',
+    largeImageText: 'GDevelop',
+  };
+
+  if (project) state.state = `Working on ${project.getName()}.`;
```

Comment:
Can you add `Working on xxx with GDevelop`? (I want "GDevelop" to be always showing).

Reply:
I changed it but not that way. I read the best practies for rich presence and it should be the shortest possible. See https://discord.com/developers/docs/rich-presence/best-practices:

![image](https://user-images.githubusercontent.com/19349038/89763412-c200e200-daf2-11ea-94b8-f8b293137ec9.png)